_posts/2023-06-06-Client-cert-auth-without-roles.adoc

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title: 'Using Certificate Based Authentication with Mutual TLS without Roles'
+date: 2023-06-06
+tags: cert-Generation server-Configuration deploying-App
+synopsis: How to deploy a simple web app on eildfly that uses mutual TLS connection and CLIENT_CERT to authenticate the client. 
```

Comment:
s/eildfly/WildFly :)

Reply:
Maybe we could reword this to something like:

How to secure a web application deployed to WildFly with mutual TLS and the CLIENT_CERT HTTP authentication mechanism.